database/engine/global_uuid_map/global_uuid_map.c

```diff
@@ -8,6 +8,46 @@ static uv_rwlock_t guid_lock;
 static uv_rwlock_t object_lock;
 static uv_rwlock_t global_lock;
 
+static void free_single_uuid(uuid_t *uuid)
+{
+    char uuid_str[37];
+    Pvoid_t *PValue, *PValue1;
+    char *existing_object;
+    Word_t  size;
+
+    uuid_unparse_lower(*uuid, uuid_str);
```

Comment:
`uuid_str` doesn't get used?

Reply:
blah.. will remove ; was part of debug code